libdazzle.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is there a reason to keep this module separately?

Reply:
Hi, at the beginning I had all the modules inside the main manifest file, than @TingPing suggested to use a script to generate and maintain the python modules and after that all modules but libdazzle where external. Than I moved libdazzle too to a separate file, for consistency.